### PR TITLE
readme improvements

### PR DESCRIPTION
**Pull request by rihp**
improvements and mode clarity for the readme!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,180 @@
...
+# Polywrap Python Client
+
+## [Polywrap](https://polywrap.io) is a developer tool that enables easy integration of Web3 protocols into any application. It makes it possible for applications on any platform, written in any language, to read and write data to Web3 protocols.
+
+
+# Working Features
+
+This MVP Python client enables the execution of **[WebAssembly](https://en.wikipedia.org/wiki/WebAssembly) Polywrappers** *(or just "wrappers")* on a python environment. It's built following the functionality of the [JavaScript Polywrap Client](https://github.com/polywrap/toolchain), which is currently more robust out and battle tested, as it has additional capabilities than this MVP.
+
+In the future, the Polywrap DAO will continue improving this Python capabilities to reach feature parity with the JS stack, including the possibiity of creating WASM wrappers with Python code.
```
Comment (Niraj-Kamdar) [inline README.md:12]: Creating a wrapper with python isn't part or the responsibility of python-client. Only thing python-client will be doing is running wrappers written in any langs

File: README.md [hunk 2]
```diff
@@ -1,67 +1,180 @@
...
+| polywrap-uri-resolvers | legacy | refactored |
+| polywrap-msgpack| yes, tested 100% | yes |
+| Subinvoke | wip | yes |
+| Wrap Manifest | WIP | yes |
+| Client Config Builder Package | wip | yes |
+| Interfaces | pending | yes |
+| e2e Tests | tbd | no |
+| Creating Plugins | tbd | yes |
+| Creating Python Wrappers | in a future version | yes |
+> TODO: Update table above according to test harness and maybe mention other wip clients (go, rust)
```
Comment (Niraj-Kamdar) [inline README.md:29]: Features and package are different things and we should only have feature list here.

- Invoke
- Subinvoke
- Interfaces
- Env
- Config builder
- Plugin
- Manifest
- Uri Resolution

File: README.md [hunk 3]
```diff
@@ -1,67 +1,180 @@
...
+
+# Getting Started:
+
+Have questions or want to get involved? Join our community [Discord](https://discord.polywrap.io) or [open an issue](https://github.com/polywrap/toolchain/issues) on Github.
+
+For detailed information about Polywrap and the WRAP standard, visit our [developer documentation](https://docs.polywrap.io/).
+
+## Pre-requisites
+
+>### Clone the repo.
```
Comment (Niraj-Kamdar) [inline README.md:40]: Don't use `>` here

File: README.md [hunk 4]
```diff
@@ -1,67 +1,180 @@
...
+>>
+================================= test session starts =================================
+platform darwin -- Python 3.10.0, pytest-7.1.3, pluggy-1.0.0
+rootdir: /Users/polywrap/pycode/polywrap/toolchain/packages/py, configfile: pytest.ini
+collected 26 items                                                                    
+
+tests/test_msgpack.py ..........................                                [100%]
+```
+
+> ### Debugging with Pytets:
```
Comment (Niraj-Kamdar) [inline README.md:111]: typo ``pytest

File: README.md [hunk 5]
```diff
@@ -70,6 +183,61 @@ Here's the `settings.json` file we are using for configuring the vscode:
...
+# What WASM wrappers can you execute today?
+
+Check these resources to browse a variety available wrappers, for DeFi, decentralised storage, and other development utilites:
+
+- [Wrappers.io](https://wrappers.io/)
+- [Polywrap Integrations Repository](https://github.com/polywrap/integrations)
+
+# Example call
+
+Calling a function of a wrapper from the python client is as simple as creating a file in the `TODO (?polywrap-client)` directory, importing the Polywrap Python Client, calling the Uri where the WASM wrapper is hosted, and specifying any required arguments.
```
Comment (Niraj-Kamdar) [inline README.md:199]: Let's have a working call with the hello world or simple subinvoke wrapper we have

File: README.md [hunk 6]
```diff
@@ -70,6 +183,61 @@ Here's the `settings.json` file we are using for configuring the vscode:
...
+    return result.result
+
+if __name__ == "__main__":
+    return get_eth_transactions(portfolio_address)
+```
+
+
+## Creating your own tests
+
+TODO: It is suggested to follow a [TDD](https://en.wikipedia.org/wiki/Test-driven_development) approach to build your own implementations.
```
Comment (Niraj-Kamdar) [inline README.md:224]: We aren't following TDD so it's not right to recommend it. You can describe other things that you see would be helpful to contributor writing test here though

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -7,28 +7,26 @@
...
 | Feature | [Python](https://github.com/polywrap/python-client) | [Javascript](https://github.com/polywrap/toolchain) | 
 | -- | -- | -- |
 | Invoke wrappers | yes | yes |
-| polywrap-asyncify | replaced with wasmtime | yes |
-| polywrap-uri-resolvers | legacy | refactored |
-| polywrap-msgpack| yes, tested 100% | yes |
-| Subinvoke | wip | yes |
-| Wrap Manifest | WIP | yes |
-| Client Config Builder Package | wip | yes |
+| Subinvoke wrappers | wip | yes |
```
Comment (Niraj-Kamdar) [inline README.md]: not wip but fully implemented, I'd suggest using emoji here would be better

✅  for fully working, ⚠️ for partially working , ❌  or 🚫 for not working

Comment (Niraj-Kamdar) [reply README.md]: Let's be less verbose and call invoke wrappers -> invoke, env configuration -> env, etc

File: README.md [hunk 8]
```diff
@@ -108,7 +106,7 @@ collected 26 items
 tests/test_msgpack.py ..........................                                [100%]
 ```
 
-> ### Debugging with Pytets:
+> ### Debugging with Pytest:
```
Comment (Niraj-Kamdar) [inline README.md:107]: Do not use `>` here. It's only used for notes.

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (rihp) [reply README.md]: updated to simple hello world for now, let me know if the syntax would be correct in this case

File: README.md (hunk 6 above)
Comment (rihp) [reply README.md]: updated

File: README.md (hunk 7 above)
Comment (rihp) [reply README.md]: Would it be worth having a column describing the feature though? I think having a brief description is useful to digest the components of the client!

Agreed on the emojis, updating now

File: README.md (hunk 1 above)
Comment (rihp) [reply README.md]: updated!

File: README.md [hunk 9]
```diff
@@ -1,67 +1,168 @@
...
+
+# Getting Started:
+
+Have questions or want to get involved? Join our community [Discord](https://discord.polywrap.io) or [open an issue](https://github.com/polywrap/toolchain/issues) on Github.
+
+For detailed information about Polywrap and the WRAP standard, visit our [developer documentation](https://docs.polywrap.io/).
+
+## Pre-requisites
+
+### `Clone the repo.`
```
Comment (Niraj-Kamdar) [inline README.md:25]: don't backquote heading

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]

**Review by Niraj-Kamdar — APPROVED** [review]
@rihp Thanks for tackling this. It looks much better now

**Pull request merged**